be/strings.po

```diff
@@ -426,6 +426,7 @@ msgid "If you are using a public computer or one you share with other people you
 msgstr ""
 
 #. i18n: keyword: MergeAccountInstructionsContinue
+#. i18n: keyword: ContinueMergeButton
```

Comment:
I don't believe this will work for showing the loc on the page (just tried it out).

Reply:
Ok. This was a suggestion of the POs editor. Do you prefer I choose only one keyword?

Reply:
I'll probably go through and create some more unified keywords for these so it works with our code! I'll close your PR but thank you for pointing out these duplicates :)